src/main/java/net/minecraftforge/fluids/FluidTank.java

```diff
@@ -1,14 +1,16 @@
-
 package net.minecraftforge.fluids;
 
+import javax.annotation.Nullable;
+
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.tileentity.TileEntity;
 
 /**
  * Reference implementation of {@link IFluidTank}. Use/extend this or implement your own.
  */
-public class FluidTank implements IFluidTank
+public class FluidTank implements IFluidTank, net.minecraftforge.fluids.capability.IFluidHandler
```

Comment:
There is a non-capability `IFluidHandler` in the same package as FluidTank so this is just the editor default. I'll import it properly.